packages/bot-web-ui/src/utils/help-content/__tests__/help-content.config.spec.ts

```diff
@@ -0,0 +1,702 @@
+import { help_content_types, help_content_config } from '../help-content.config';
+
+describe('help_content_config', () => {
+    const { TEXT, IMAGE, BLOCK, EXAMPLE } = help_content_types;
+    const publicPath = '/my-public-path';
+    const config = help_content_config(publicPath);
+
+    it('should set webpack_public_path and return configuration object', () => {
+        expect(window.__webpack_public_path__).toBe(publicPath);
+    });
+
+    describe('procedures_ifreturn', () => {
+        it('should have 4 items', () => {
+            expect(config.procedures_ifreturn).toHaveLength(4);
+        });
+
+        it('should have the correct types and example_id', () => {
+            expect(config.procedures_ifreturn[0]).toEqual({ type: TEXT });
+            expect(config.procedures_ifreturn[1]).toEqual({ type: BLOCK });
+            expect(config.procedures_ifreturn[2]).toEqual({ type: TEXT });
+            expect(config.procedures_ifreturn[3]).toEqual({ type: EXAMPLE, example_id: 'if-return' });
+        });
+    });
+
+    describe('math_arithmetic', () => {
+        it('should have 8 items', () => {
+            expect(config.math_arithmetic).toHaveLength(8);
+        });
+
+        it('should have the correct types', () => {
+            expect(config.math_arithmetic[0]).toEqual({ type: TEXT });
+            expect(config.math_arithmetic[1]).toEqual({ type: BLOCK });
+            expect(config.math_arithmetic[2]).toEqual({ type: TEXT });
+            expect(config.math_arithmetic[3]).toEqual({ type: TEXT });
+            expect(config.math_arithmetic[4]).toEqual({ type: TEXT });
+            expect(config.math_arithmetic[5]).toEqual({ type: TEXT });
+            expect(config.math_arithmetic[6]).toEqual({ type: TEXT });
+            expect(config.math_arithmetic[7]).toEqual({ type: TEXT });
+        });
+    });
+
+    describe('math_single', () => {
+        it('should have 9 items', () => {
+            expect(config.math_single).toHaveLength(9);
+        });
+
+        it('should have the correct types', () => {
+            expect(config.math_single[0]).toEqual({ type: TEXT });
+            expect(config.math_single[1]).toEqual({ type: BLOCK });
+            expect(config.math_single[2]).toEqual({ type: TEXT });
+            expect(config.math_single[3]).toEqual({ type: TEXT });
+            expect(config.math_single[4]).toEqual({ type: TEXT });
+            expect(config.math_single[5]).toEqual({ type: TEXT });
+            expect(config.math_single[6]).toEqual({ type: TEXT });
+            expect(config.math_single[7]).toEqual({ type: TEXT });
+            expect(config.math_single[8]).toEqual({ type: TEXT });
+        });
```

Comment:
Can we write all the type in one array and loop them to write one expect only instead of so many expects

```
it('should have the correct types', () => {
        const expectedTypes = [
            { type: TEXT },
            { type: BLOCK },
            { type: TEXT },
            { type: TEXT },
            { type: TEXT },
            { type: TEXT },
            { type: TEXT },
            { type: TEXT },
            { type: TEXT }
        ];

        expectedTypes.forEach((expectedType, index) => {
            expect(config.math_single[index]).toEqual(expectedType);
        });
    });
```

Reply:
@maryia-matskevich-deriv

Reply:
@maryia-matskevich-deriv something like the above example

Reply:
all objects from the function will be copied in this way, I'm not sure that it will be a good approach, so I prefer to use another option... but we can go with any option 
<img width="714" alt="Screenshot 2024-03-13 at 12 14 58" src="https://github.com/binary-com/deriv-app/assets/103181650/20b80d0d-aea0-4ad5-875f-982aa60329d5">

Reply:
@maryia-matskevich-deriv since this a config file, we don't need to test so extensively with each type. It is too long for a file which doesn't have any function. If we still want to test then maybe we can test the main objects only.

Reply:
the file has been removed